homeassistant/components/keenetic_ndms2/device_tracker.py

```diff
@@ -29,7 +29,9 @@
         vol.Required(CONF_USERNAME): cv.string,
         vol.Required(CONF_PORT, default=DEFAULT_PORT): cv.port,
         vol.Required(CONF_PASSWORD): cv.string,
-        vol.Required(CONF_INTERFACE, default=DEFAULT_INTERFACE): cv.string,
+        vol.Required(CONF_INTERFACES, default=DEFAULT_INTERFACES): vol.All(
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>